cli/src/command/account_completion.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::borrow::Cow;
+
+use colored::Colorize;
+use rustyline::{
+    completion::Completer, highlight::Highlighter, hint::HistoryHinter, Completer, Context, Helper, Hinter, Validator,
+};
+
+#[derive(Default)]
+pub struct AccountCompleter;
+
+const ACCOUNT_COMMANDS: &[&str] = &[
+    "accounts",
+    "addresses",
+    "balance",
+    "burn-native-token",
+    "burn-nft",
+    "claim",
+    "claimable-outputs",
+    "clear",
+    "consolidate",
+    "create-alias-output",
+    "create-native-token",
+    "destroy-alias",
+    "destroy-foundry",
+    "exit",
+    "faucet",
+    "melt-native-token",
+    "mint-native-token",
+    "mint-nft",
+    "new-address",
+    "node-info",
+    "output",
+    "outputs",
+    "send",
+    "send-native-token",
+    "send-nft",
+    "switch",
+    "sync",
+    "transaction",
+    "transactions",
+    "tx",
+    "txs",
+    "unspent-outputs",
+    "vote",
+    "stop-participating",
+    "participation-overview",
+    "voting-power",
+    "increase-voting-power",
+    "decrease-voting-power",
+    "voting-output",
+    "help",
+];
+
+impl Completer for AccountCompleter {
+    type Candidate = String;
```

Comment:
Nit but it doesn't look like this needs to be a String

Reply:
Done